### PR TITLE
Cirrus: Update to Fedora 32 proper

### PR DESCRIPTION
**Pull request by cevich**
Now that it's officially released, update to it from the beta.  Also
(and significant), adjust the SELinux context of the GCP metadata
service.  Add a comment to the code explaining why this is necessary.

Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

**Comment by cevich** [comment]
@baude @mheon PTAL, this is ready to go and only a small change.  These new Fedora images (including in_podman) are also used by https://github.com/containers/buildah/pull/1971

File: .cirrus.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ env:
     UBUNTU_NAME: "ubuntu-19"
     PRIOR_UBUNTU_NAME: "ubuntu-18"
 
-    _BUILT_IMAGE_SUFFIX: "libpod-6220812239765504"
+    _BUILT_IMAGE_SUFFIX: "libpod-6224667180531712"  # From the packer output of 'build_vm_images_script'
```
Comment (TomSweeneyRedHat) [inline .cirrus.yml:42]: :+1:  to the comment!

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by cevich** [comment]
Thanks guys.

**Pull request merged**